tests/integration/expected_test_results/api_v3_taxonomy_suggestions/packaging-shapes-cc-fr-categories-en-yogurts.json

```diff
@@ -2,31 +2,7 @@
    "errors" : [],
    "status" : "success",
    "suggestions" : [
-      "Pot",
-      "Seal",
```

Comment:
Any idea why this test for packaging and other tests like [categories-string-fr-tart-get-synonyms.json](https://github.com/openfoodfacts/openfoodfacts-server/pull/12383/files#diff-5a1b04492da94d05504c582bb8e52b82ec4052bd8fae7128dda08bca22689df3) for categories are affected by the change? I feel like this part of the snapshots shouldn't need to be updated for the unwanted ingredients API?